modules/local-caching-classloader/src/test/java/org/apache/accumulo/classloader/lcc/LocalCachingContextClassLoaderFactoryTest.java

```diff
@@ -224,10 +224,8 @@ public void testInitialInvalidJson() throws Exception {
         def.toJson().substring(0, 4));
     final URL invalidDefUrl = new URL(fs.getUri().toString() + invalid.toUri().toString());
 
-    ContextClassLoaderException ex = assertThrows(ContextClassLoaderException.class,
+    assertThrows(ContextClassLoaderException.class,
         () -> FACTORY.getClassLoader(invalidDefUrl.toString()));
```

Comment:
Current change is fine, but it would probably be good to still ensure the cause is a JSON problem. Could do something like the following
```suggestion
    Throwable exception = assertThrows(ContextClassLoaderException.class,
() -> FACTORY.getClassLoader(invalidDefUrl.toString()));
    // ensure json is somewhere in the exception
    outer: while (exception != null) {
      for (var trace : exception.getStackTrace()) {
        if (trace.toString().contains("com.google.gson")) {
          break outer;
        }
      }
      exception = exception.getCause();
    }
    assertNotNull(exception);
```